src/groups/mqb/mqbcfg/mqbcfg.xsd

```diff
@@ -239,9 +241,11 @@
         heartbeatIntervalMs..:
             How often (in milliseconds) to check if the channel received data,
             and emit heartbeat.  0 to globally disable.
-       useNtf...............:
+        useNtf...............:
             Use the new NTF based TCP transport library instead of
             the existing one based on BTE
+         tls.................:
```

Comment:
Possible to rename this flag to `useTls` similar to `useNtf`.

Reply:
Yeah `useTls` is probably better.  Also, should this field go inside `TlsConfig` complex type?

Reply:
> Yeah `useTls` is probably better. Also, should this field go inside `TlsConfig` complex type?

It's a good question.
We have several possibilities.

1. Current structure, `useTls` and `tlsConfig` settings are placed in different paths.
2. `useTls` is a part of `tlsConfig`.
3. `useTls` is not a part of `tlsConfig`, but they both are placed near each other in `TcpInterfaceConfig`.
4. Implicit approach: no `useTls` flag, optional `tlsConfig` presence is a marker that TLS should be used.

2 and 3 are good for me.